src/middleware/devtools.ts

```diff
@@ -174,7 +175,7 @@ export const devtools =
       if (!isRecording) return
       extension.send(
         nameOrAction === undefined
-          ? { type: devtoolsOptions.anonymousActionType ?? 'anonymous' }
+          ? { type: devtoolsOptions.anonymousActionType || 'anonymous' }
```

Comment:
I suppose falsy check instead of nullish check is enough (can't imagine if anyone want empty string for action type). There's no other code using nullish check in this repo. So, want to make it consistent.